netket/graph/symmetry.py

```diff
@@ -80,6 +81,61 @@ def remove_duplicates(self, *, return_inverse=False):
         else:
             return group
 
+    def inverse(self):
+        """
+        Returns reordered SymmGroup where the each element is the inverse of
+        the original symmetry element. If :code:`g = self[element]` and :code:`h = self.inverse()[element]`,
+        then :code:`gh = product(g, h)` will act as the identity on the sites of the graph, i.e., :code:`np.all(gh(sites) == sites)`.
+
+        """
+
+        automorphisms = self.to_array()
+        n_symm = len(automorphisms)
+        inverse = np.zeros([n_symm], dtype=int)
+        automorphisms = np.array(automorphisms)
+        for i, perm1 in enumerate(automorphisms):
+            for j, perm2 in enumerate(automorphisms):
+                perm_sq = perm1[perm2]
+                if np.all(perm_sq == np.arange(len(perm_sq))):
+                    inverse[i] = j
+
+        return SymmGroup([self.elems[i] for i in inverse], self.graph)
```

Comment:
I wonder which is better: Returning a `SymmGroup` here or just the indices of the inverse elements (i.e., `inverse` directly). The indices could be a bit more generally useful and are harder to reconstruct from `.elems` later.

Reply:
I've vacilated on this. I'm happy to return the indices

Reply:
Yes, let's return the indices.

Reply:
It looks to me this was not reverted to indices?

Reply:
nevermind, i was on an old diff